index.html

```diff
@@ -1,17 +1,19 @@
 ---
 layout: default
+title: home
 description: Landing
 keywords:
 author:
 ---
 
-<div>
-	<img src="images/centeredorange.png" width="35%" class="mdl-grid mdl-cell--middle">
-</div>
-
-<div class="mdl-color-text--white mdl-typography--text-center">
-	<h1>CUCTF 2020</h1>
-	<h3>October 03, 2020</h3>
-	<h3>405 S Palmetto Blvd, Clemson, SC</h3>
-	<h3>Hosted by CU Cyber</h3>
+<img src="public/images/cu_shield.png" class="logo">
+<div class="container white-text">
+  <h2 class="center">Welcome to <strong>CUCTF</strong>.</h2>
+  <p class="json">
+  <br>
+  <strong>"event"</strong>: <strong>"CUCTF 2020"</strong> <span class="info">/* Hosted by CU Cyber */</span><br>
+  <strong>"date"</strong>: <strong>"03 October 2020"</strong><br>
+  <strong>"location"</strong>: <strong>"405 S Palmetto Blvd, Clemson"</strong><br>
+  <strong>"Version"</strong>: <strong>2.0</strong>
```

Comment:
I like the formatting, but the capitalization of the first element in each line should be consistent.